docs/trusted-resources.md

```diff
@@ -91,3 +91,36 @@ metadata:
 data:
   publickeys: "/etc/verification-secrets/cosign.pub, /etc/verification-secrets/cosign2.pub"
 ```
+
```

Comment:
I noticed that this doc seems like it's trying to delegate to TEP-0091 to describe the design - i suggest you describe the functionality here as well explicitly, even if it feels like duplicating the TEP. There are a couple of reasons for this:

* These docs are living docs and will get updates as changes are made, whereas TEPs will not (there might be new TEPs, and also issues and PRs that change the design and functionality)
* The audience for each doc is different: TEPs are primarily for folks who work on Tekton to understand proposed changes, and these docs are for users of Tekton to understand how to use features